plugin/postgres/plugin.go

```diff
@@ -154,8 +163,21 @@ func (p PostgresPlugin) Backup(endpoint ShieldEndpoint) error {
 	}
 
 	setupEnvironmentVariables(pg)
-	cmd := fmt.Sprintf("%s/pg_dumpall -c --no-password", pg.Bin)
-	DEBUG("Executing: `%s`", cmd)
+
+	// If user specified specific database to backup
+	db_name := pg.Database
+
+	cmd := ""
+	// Check if db_name is valid
+	if db_name != "" {
+		// Run dump all on the specified db
+		cmd = fmt.Sprintf("%s/pg_dump %s -c --no-password", pg.Bin, db_name)
+		DEBUG("Executing: `%s`", cmd)
+	} else {
+		// Else run dump on all
+		cmd = fmt.Sprintf("%s/pg_dumpall -c --no-password", pg.Bin)
+		DEBUG("Executing: `%s`", cmd)
```

Comment:
These two DEBUG statements can go outside of the conditional blocks, since they are identical in both cases.